lambeq/backend/drawing/tikz_backend.py

```diff
@@ -38,27 +38,64 @@ def __init__(self, use_tikzstyles: bool = False):
         self.nodes: dict[tuple[float, float], int] = {}
         self.nodelayer: list[str] = []
         self.edgelayer: list[str] = []
+        self.label_layer: list[str] = []
         self.max_width: float = 0
 
-    @staticmethod
-    def format_color(color: str) -> str:
+    def format_color(self, color: str) -> str:
```

Comment:
@Ragunath1729 what is the reason for splitting this?

Reply:
the wire colouring is different logic than box colouring. Boxes uses a tuple logic, while the wires has noun_id. and we get  the hex_code directly at before drawing. the common portion is to convert the hex-> RGB , which could be made redundant. but thought better to reuse the code by refactoring.